pycqed/instrument_drivers/meta_instrument/LutMans/mw_lutman.py

```diff
@@ -222,17 +222,21 @@ def generate_standard_waveforms(
             elif waveform['type'] == 'square':
                 # Using a slightly different construction as above
                 # as the call signatures of these functions is different.
-                if 'sq_amp' in self.parameters:
-                    self._wave_dict[idx] = wf.mod_square(
-                        amp=self.sq_amp(), length=self.mw_gauss_width()*4,
-                        f_modulation=self.mw_modulation(),  phase=0,
-                        motzoi=0, sampling_rate=self.sampling_rate())
-                elif 'sq_G_amp' in self.parameters:
+                # Apperently the VSM LutMan has both parameters, so make sure
+                # we detect on the one only available in the VSM. Otherwise, we
+                # won't get the needed four waveforms.
+                if 'sq_G_amp' in self.parameters:
                     self._wave_dict[idx] = wf.mod_square_VSM(
                         amp_G=self.sq_G_amp(), amp_D=self.sq_D_amp(),
                         length=self.mw_gauss_width()*4,
                         f_modulation=self.mw_modulation(),
                         sampling_rate=self.sampling_rate())
+                elif 'sq_amp' in self.parameters:
+                    print('Hello my friend!')
+                    self._wave_dict[idx] = wf.mod_square(
+                        amp=self.sq_amp(), length=self.mw_gauss_width()*4,
+                        f_modulation=self.mw_modulation(),  phase=0,
+                        motzoi=0, sampling_rate=self.sampling_rate())
```

Comment:
We should use the snippet of sq_amp and sq_G_amp oof the current version of develop as that is known to work. These changes (including the print statement) appear to be the consequence of a wrong merge.

Reply:
@AdriaanRol are you really sure about this?

Currently, the `AWG8_MW_LutMan` defines the `sq_amp` parameter. The `AWG8_VSM_MW_LutMan` class is derived from that class and calls its method for adding parameters, such that it also gets the `sq_amp` parameter in addition to `sq_G_amp` and `sq_D_amp`. As a consequence, when the waveforms are generated in the VSM version of the class, it will see the `sq_amp` parameter and only generate a two-channel waveform, which causes problems with the tests.

Has this issue been fixed somewhere else, because I cannot understand how it's supposed to pass the test without my (or a similar) modification?